.helmignore

```diff
@@ -21,12 +21,18 @@
 .gitignore
 *.tmp
 *~
+Jenkinsfile
+mvn_dependency_tree.txt

```

Comment:
The mvn_dependency_tree.txt should continue to be checked in, (in my humble opinion) - otherwise we will not be able to review dependency changes as part of the commit.

Reply:
this is a helm ignore file, not a git ignore file.